src/main/java/org/fundacion/jala/converter/models/facade/ConverterFacade.java

```diff
@@ -15,15 +15,19 @@
 import org.fundacion.jala.converter.service.AudioConverter;
 import org.fundacion.jala.converter.service.VideoConverter;
 
+/**
+ * This class calls facade of converter.
+ */
 public class ConverterFacade {
 
     private ConverterFacade() {
     }
 
     /**
      * Converts an audio file.
```

Comment:
The description of this method seems to be incorrect. The name of the method is getAudioConverter

Reply:
ok It was solved